src/AddTodoForm.jsx

```diff
@@ -17,16 +18,11 @@ function AddTodoForm({ onAddTodo }) {
 
   return (
     <form onSubmit={handleAddTodo}>
-      <label htmlFor="todoTitle">Title</label>
-      <input
-        id="todoTitle"
-        name="title"
-        value={todoTitle}
-        onChange={handleTitleChange}
-      ></input>
-      <button>Add</button>
+      <InputWithLabel todoTitle={todoTitle} onTitleChange={handleTitleChange}>
```

Comment:
You're using the wrong name for the change handler prop, so adding a todo isn't working correctly